Berlin/BerOrOct59.xml

```diff
@@ -26,43 +30,250 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <repository ref="INS0333SBB"/>
                   <collection>Ms. orient. oct.</collection>
 
-                  <idno>SBP, Ms. orient. oct. 59</idno>
+                  <idno facs="https://content.staatsbibliothek-berlin.de/dc/863180108/manifest">SBP, Ms. orient. oct. 59</idno>
                </msIdentifier>
                <msPart xml:id="p1">
                <msIdentifier> <idno>SBP, Ms. orient. oct. 59, p1</idno>
              </msIdentifier>
              <msContents>
                <msItem xml:id="p1_i1">
                  <locus from="1r" to="4v"></locus>
-                 <title type="complete" ref="LIT2509Weddas"></title>
-                 <textLang mainLang="lat"> </textLang>
+                   <title> Description of the content by Hiob Ludolf </title>
+                 <textLang mainLang="la"> </textLang>
+                 <textLang otherLangs="gez"> </textLang>
                  </msItem>
                  </msContents>
+
+                 <physDesc>
+                    <objectDesc form="Codex">
+                       <supportDesc>
+                        <support>
+                           <material key="paper"/>
+                            </support>
+
+                    <foliation>
+                        <list>
+                         <item resp="p1">
+                          <locus from="1r" to="4r"/>
+                          Foliation in pencil in the upper right corner.
+                           </item>
+                        </list>
+                     </foliation>
+                     </supportDesc>
+                   </objectDesc>
+
+                 <handDesc>
+                    <handNote xml:id="h2" script="Latin" corresp="p1_i1">
```

Comment:
I do not see a h1, perhaps simply copy pasting mistake?

Reply:
please use # for @corresp, to point to an id https://betamasaheft.eu/Guidelines/?q=handNote&id=handDescription

Reply:
found h1, much below. so, all good for that.